bundles/GraphicServiceOrder/Resources/views/createOrderSubmitted.html.twig

```diff
@@ -28,8 +31,8 @@
   <div class="mb-2">
     <span class="label"><strong>{{ 'service_order_form.job_description.files.label'|trans }}:</strong></span>
     <ul>
-      {% if form_data.files is defined %}
-        {% for file in form_data.files %}
+      {% if order.files is defined %}
```

Comment:
`<ul>…</ul>` should be moved inside `{% if … %}…{% endif %}` and using `order.files|default(false)` will also check that `order.files` is not empty.

Reply:
fixed